Updating build index: /Users/runner/conda/conda-bld

TEST START: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpy3n9_apd/osx-64/semidbm-0.5.1-py36_1.tar.bz2
Adding in variants from internal_defaults
Reloading output folder: /Users/runner/conda/conda-bld
Mamba failed to solve:
 - semidbm 0.5.1.* py39_1
 - python
 - python

with channels:
 - file:///var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpy3n9_apd

The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested semidbm 0.5.1.* py39_1
- 

Leaving build/test directories:
  Work:
 /Users/runner/conda/conda-bld/work 
  Test:
 /Users/runner/conda/conda-bld/test_tmp 
Leaving build/test environments:
  Test:
source activate  /Users/runner/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla 
  Build:
source activate  /Users/runner/conda/conda-bld/_build_env 


